### ISSUE TITLE
List Applications which use given table.

### ISSUE THREAD

**Issue by Imbruced** [AbsaOSS/spline#746]
## Question

Hi, Is it possible to list all the **application names** which are using given table/ hdfs location ? For example I have Hive table **test.table** and my goal is to show applications which using it.

### input 

test.table

### expected result
```
+----------------+-------------------+----------+
|application_name|     application_id|    origin|
+----------------+-------------------+----------+
|     data-merger|application_1214234|test.table|
| data-aggregator|application_5496876|test.table|
+----------------+-------------------+----------+

```

**Comment by wajda** [comment]
There is no such feature in the UI yet, but you can do it by querying ArangoDB directly
For example:
```aql
FOR ds IN dataSource
    FILTER LIKE(ds.uri, "%test.table%")
    FOR execPlan IN 1..1 INBOUND ds depends // or `affects` or `depends, affects` if you want both
        COLLECT appName=execPlan.extra['appName']
        RETURN appName
```

**Comment by Imbruced** [comment]
Thank you so much ! :)

**Comment by wajda** [comment]
Will be implemented in #804
